forms-flow-components/src/components/CustomComponents/NoDataFound.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+export const NoDataFound: React.FC = () => {
+  const { t } = useTranslation();
+
+  return (
+    <tbody className="no-data-body">
+      <tr>
+        <td colSpan={2} className="no-data-container">
+          <span className="no-data-text">{t("Nothing is found. Please try again.")}</span>
```

Comment:
It’s a good idea to add a data-test-id attribute to the `<span> `element, as it will be helpful for automation testing.

Reply:
Updated.